debugProtocol.json

```diff
@@ -3190,6 +3359,50 @@
 					"description": "Details of the exception contained by this exception, if any."
 				}
 			}
+		},
+
+		"DisassembledInstruction": {
+			"type": "object",
+			"description": "Represents a single disassembled instruction.",
+			"properties": {
+				"address": {
+					"type": "string",
+					"description": "The address of the instruction.  Treated as a hex value if prefixed with '0x', or as a decimal value otherwise."
+				},
+				"instructionBytes": {
+					"type": "string",
+					"description": "Raw bytes representing the instruction and its operands, in an implementation-defined format."
+				},
+				"instruction": {
```

Comment:
Clarification: So this can be really any string (raw). including a comment description?  Examples:
```
 80242f2:	4b08      	ldr	r3, [pc, #32]	; (8024314 <Cy_Flash_RAMDelay+0x60>)
 8024308:	f8d3 351c 	ldr.w	r3, [r3, #1308]	; 0x51c
 80242cc:	daf5      	bge.n	80242ba <Cy_Flash_RAMDelay+0x6>
```
I am not just focused on ARM. Just an example I have at the moment. I can find similar things with MS/Intel stuff.

Reply:
Yup, if an adapter wanted to return something like `ldr	r3, [pc, #32]	; (8024314 <Cy_Flash_RAMDelay+0x60>)` that would be entirely fine.